pom.xml

```diff
@@ -130,7 +130,13 @@
         <dependency>
             <groupId>org.sourcelab</groupId>
             <artifactId>kafka-connect-client</artifactId>
-            <version>3.1.0</version>
+            <version>4.0.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
Added guava as a dependency, because kafka-connect-client new version removed it from it's dependency.